packages/@webex/plugin-meetings/src/index.ts

```diff
@@ -3,9 +3,13 @@ import {registerPlugin} from '@webex/webex-core';
 
 import Meetings from './meetings';
 import config from './config';
+import {LocusRetryStatusInterceptor} from './interceptors';
 
 registerPlugin('meetings', Meetings, {
   config,
+  interceptors: {
+    LocusRetryStatusInterceptor: LocusRetryStatusInterceptor.create,
```

Comment:
question: how does this actually apply the interception to the locus service URL?

Reply:
When the meeting plugin is mounted, this interceptor should be mounted as well.

This then gets applied to all requests, see the `on{phase}` methods written within the child-interceptor's class.